### PR TITLE
Changes needed to make CircleCI use snapshot repos

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Comment by geoand** [comment]
If you guys agree with this PR, then I will merge it and issue a change to the other boosters directly on the master branch

**Comment by geoand** [comment]
The tests look they failed because I mistakenly canceled them from the CircleCI dashboard :(

**Comment by geoand** [comment]
I restarted the tests manually and the bot here picked up the changes :+1:

File: .circleci/settings.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+  <profiles>
+    <profile>
+      <id>snapshots</id>
```
[the remark this answers is not in the document]
Comment (geoand) [reply .circleci/settings.xml]: It's active be default :)

[the remark this answers is not in the document]
Comment (geoand) [reply .circleci/settings.xml]: https://github.com/snowdrop/spring-boot-http-secured-booster/blob/95f6b0b5cdf6ad021a898c4e7f1ea452d805f438/.circleci/settings.xml#L8

[the remark this answers is not in the document]
Comment (metacosm) [reply .circleci/settings.xml]: D'oh! /me needs coffee!

[the remark this answers is not in the document]
Comment (metacosm) [reply .circleci/settings.xml]: Though, why did you create a profile then?

[the remark this answers is not in the document]
Comment (geoand) [reply .circleci/settings.xml]: We can't (to my knowledge) add the repositories anywhere else in `settings.xml` file. @gytis is my assumption correct?

[the remark this answers is not in the document]
Comment (metacosm) [reply .circleci/settings.xml]: D'oh^2! When is that coffee getting here?

[the remark this answers is not in the document]
Comment (geoand) [reply .circleci/settings.xml]: Should I send a Monster Energy your way? :P

https://www.trademarksandbrandsonline.com/media/image/jonathan-weiss-shutterstock-inc-monster-energy-.jpg

[the remark this answers is not in the document]
Comment (metacosm) [reply .circleci/settings.xml]: At this point I guess I need all the help I can get! :D

**Pull request merged**